src/components/header/index.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getTeams, getSelectedTeam } from '../../actions/actions.js';
+
+import NavMenu from './navMenu';
+
+import bell from '../../assets/icons/notification-bell.svg';
+import settingsIcon from '../../assets/icons/settings-icon.svg';
+import kobe from '../../assets/imgs/kobe.png';
+
+import styles from './index.module.scss';
+
+class Header extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = { 
+            selectedTeamName: null, 
+            selectedTeamTricode: null,
+
+            // NOTE: when the teams are mapped, the raptors are index 27, hence the default number
+            selectedTeam: 27
+        }  
+    }
+
+    componentDidMount() {
+        this.props.getTeams();
+    }
+
+    onSelectChange = (e) => {
+        const { teams, getSelectedTeam } = this.props;
+
+        this.setState({ 
+            selectedTeamName: teams[e.target.value].ttsName, 
+            selectedTeamTricode: teams[e.target.value].tricode,
+            selectedTeam: e.target.value, 
+        });
+
+        getSelectedTeam(teams[e.target.value]);
+    }
+
+    render() {
+        const { selectedTeamTricode, selectedTeam } = this.state;
+        const { teams, selectedTeamColor } = this.props;
+
+        // NOTE: I've used some in-line styling to access the 'selected team color' variable
```

Comment:
`// Used in-line styling to access 'selected team color' variable`

It might seem like a nitpick, but when leaving comment no need to prefix it with `NOTE:`, that's understood. Also make it  concise and remove any personal words like `I've` or `Mine`, this code should be neutral. Just a tip going forward.

That being said, it's good to leave notes like this when necessary. The code itself should be self-explanatory, but it's always good to comment whenever there is something that you think needs further clarification.